File: Bucles/Ejercicio13-Bucles.cpp
/* 13. Hacer un programa que realice la serie de Fibonacci:
Fibonacci -> 0,1,1,2,3,5,8,13,....,n */

#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    int n, x=0, y=1, z=1;
    cout<<"Digite el numero de elementos: "; cin>>n;

    cout<<"\n0 1 ";
    for (int i=1; i<n; i++){
        z  = x + y;
        cout<<z<<" ";
        x = y;
        y = z;
    }
    cout<<"\n";
    system("pause");
    return 0;
}

File: Comparativa Python/Ejercicio13-Bucles.py
"""13. Hacer un programa que realice la serie de Fibonacci:
Fibonacci -> 0,1,1,2,3,5,8,13,....,n """

x = 0
y = 1
z = 1

n = int(input('Digite número de iteraciones: '))
print('0, 1', end="")
for i in range(1, n + 1, 1):
    z = x + y
    print(', {}'.format(z), end="")
    x = y
    y = z
